nifi-api/src/main/java/org/apache/nifi/components/ValidationContext.java

```diff
@@ -21,9 +21,13 @@
 import org.apache.nifi.controller.ControllerService;
 import org.apache.nifi.controller.ControllerServiceLookup;
 import org.apache.nifi.expression.ExpressionLanguageCompiler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
This introduces a dependency on `slf4j-api` in `nifi-api`, which is coming through right now from `slf4j-simple`. As all of the logging statements are for debugging, it seems like the most straightforward option is to remove the debug logging.